src/Pension/Pensionsberechnung.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Finanzrechner\Pension;
+
+/**
+ * Class BeamtenPensionRechner
+ * @package Finanzrechner\Pension
+ */
+class Pensionsberechnung
+{
+    private float $pensionsbetrag;
+
+    public function getPensionsbetrag(): float
+    {
+        return $this->pensionsbetrag;
+    }
+
+    public function isMindestruhegehalt(): bool
+    {
+        return $this->isMindestruhegehalt;
+    }
+
+    private bool $isMindestruhegehalt;
+
+    public function __construct(float $pensionsbetrag, bool $isMindestruhegehalt)
+    {
+        $this->pensionsbetrag      = $pensionsbetrag;
+        $this->isMindestruhegehalt = $isMindestruhegehalt;
+    }
```

Comment:
- Der Konstruktor einer Klasse steht immer ganz oben vor allen anderen Methoden. Das ist quasi der Einsteigspunkt oder die Überschrift, wie in einem Buchkapitel. Da werden grundlegende Dinge festgelegt, die bei jeder Instanz diesr Klasse IMMER gleich ablaufen.
- Die Deklaration (und z.T. auch Initialisierung) der Instanzvariablen findet noch vor dem Konstruktor statt. So sieht man schnell, welche Variablen es gibt und welchen Typ sie haben.
- Erst dann kommen die Methoden. Zuerst die public-Methoden, weil diese die öffentlichen Schnittstelle einer Klasse darstellen und somit am wichtigsten sind. Danach die private Methoden (die hast du hier nicht, aber so grundsätzlich)